### PR TITLE
Update firmware.csv

### PR DESCRIPTION
**Pull request by ricardojohansson**
Updated firmware on spare logger prior to deployment

### CODE REVIEW

File: install/firmware.csv [hunk 1]
```diff
@@ -42,7 +42,8 @@ Campbell Scientific,CR6,17444,CR6.Std.11.01,2022-03-10T00:00:00Z,2023-11-29T00:0
 Campbell Scientific,CR6,17444,CR6.Std.12.02,2023-11-29T00:00:01Z,9999-01-01T00:00:00Z,
 Campbell Scientific,CR6,17445,CR6.Std.11.01,2022-03-10T00:00:00Z,2023-01-29T23:59:00Z,
 Campbell Scientific,CR6,17445,CR6.Std.11.02,2023-01-30T00:00:00Z,9999-01-01T00:00:00Z,
-Campbell Scientific,CR6,18430,CR6.Std.11.01,2022-03-10T00:00:00Z,9999-01-01T00:00:00Z,
+Campbell Scientific,CR6,18430,CR6.Std.11.01,2022-03-10T00:00:00Z,2024-09-03T23:59:00Z,
```
Comment (EvanCGNS) [inline install/firmware.csv:45]: `2024-09-03T23:59:00Z` should be `2024-09-02T23:59:00Z`

Comment (ricardojohansson) [reply install/firmware.csv:45]: Good catch!

**Comment by salichon** [comment]
This  PR is blocked  and shall be resolved

**Comment by elidana** [comment]
have pushed the change to fix the typo on end date (good catch @EvanCGNS !)

**Review by elidana — APPROVED** [review]

**Pull request merged**